README.md

```diff
@@ -2,43 +2,62 @@
 
 This is the repo for template deployment schemas hosted under `https://schema.management.azure.com/schemas`. Please see below for information on contributing and publishing updated schemas.
 
-## Submitting a PR
-* Ensure that any new resource types that you are adding have been added to the following top-level template schemas:
-  * [schemas/2014-04-01-preview/deploymentTemplate.json](/schemas/2014-04-01-preview/deploymentTemplate.json)
-  * [schemas/2015-01-01/deploymentTemplate.json](/schemas/2015-01-01/deploymentTemplate.json)
-  * [schemas/2019-04-01/deploymentTemplate.json](/schemas/2019-04-01/deploymentTemplate.json)
-* If your schema has been manually generated, please ensure you include appropriate tests in [tests](/tests/)
-* If adding a new resource type, please add examples to the templates in [tools/templateTests](/tools/templateTests/)
-* Ensure that the test suite passes (see [Tests](#tests))
+## Updating Schemas
+There are two processes for updating schemas:
+1. **Nightly autogeneration**: Schemas are automatically generated from definitions in the [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo. This requires [onboarding](/generator/README.md#onboarding-teams-to-autogeneration-pipeline) on a per-provider basis.
```

Comment:
"Daily autogeneration" - meaning every day rather than "Nightly"? Just a suggestion

Reply:
Good point - makes sense to use consistent terminology. I've fixed this.